packages/https-proxy/lib/ca.coffee

```diff
@@ -135,7 +135,7 @@ class CA
     sn
 
   generateCA: ->
```

Comment:
need a way to ensure that the CA is recreated, older usersof cypress will have it cached